src/test/java/study/StringTest.java

```diff
@@ -0,0 +1,60 @@
+package study;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+class StringTest {
+
+    @Test
+    @DisplayName("split() 메소드를 테스트한다.")
+    void split() {
+        // given
+        String data = "1,2";
+
+        // when
+        String[] split = data.split(",");
+
+        // then
+        assertAll(
+                () -> assertThat(split).contains("1"),
+                () -> assertThat(split).contains("2"),
+                () -> assertThat(split).containsExactly("1", "2")
+        );
+    }
+
+    @Test
+    @DisplayName("substring() 메서드를 테스트한다.")
+    void substring() {
+        // given
+        String data = "(1,2)";
+
+        // when
+        String substring = data.substring(1, data.length() - 1);
+
+        // then
+        assertThat(substring).isEqualTo("1,2");
```

Comment:
```suggestion
        String actual = data.substring(1, data.length() - 1);
        expected = "1,2";

        assertThat(actual).isEqualTo("1,2");
```

이런건 어떨까요? 검증하고자 하는 값의 변수에 대한 이름을 걱정할것 없이
테스트 코드에서의 의미를 일관되게 부여하는것이죠 😄

Reply:
substring 보다 actual로 표현하는 것이 가독성 측면에서 훨씬 좋은것 같습니다 :)